src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export default function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age ? <p className="Person__age">I am {person.age}</p> : null}
+      {person.isMarried ? (
+        <p className="Person__partner">
+          {person.partnerName} is my
+          {person.sex === 'f' ? ' husband' : ' wife'}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
nested ternary operators are a bad practice


decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```